packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -180,6 +180,15 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     );
   }
 
+  protected okToAddMore(): boolean {
```

Comment:
minor suggestion; `canAddItems` may be a better name. ("more" of what? "ok to" is also inconsistent)